lib/external.py

```diff
@@ -215,3 +217,31 @@ def check_django(self, machine, manage_py, exception=True):
 
             return ret == 0 # 0 - no issues, 1 - problems
 
+    @contextmanager
+    def connect_rds(self):
```

Comment:
I would locate this file after `ExternalCalls.vault` or `ExternalCalls.tunnel` so that functionality is grouped together